base/tools/src/main/java/com/netscape/cmstools/kra/KRAKeyRetrieveCLI.java

```diff
@@ -52,18 +52,10 @@ public void createOptions() {
         option.setArgName("Passphrase");
         options.addOption(option);
 
-        option = new Option(null, "input-format", true, "Input format: xml (default), json");
-        option.setArgName("format");
-        options.addOption(option);
-
```

Comment:
Will this cause existing scripts/automation that use `--input-format json` to fail? If so we probably should keep the option, but drop the `xml` and make `json` the default. If it's not causing a problem then this is fine.

Reply:
Good point, I have reinstated the option defaulting to `JSON`. If `XML` is requested you will get errors, but requesting `JSON` is now backwards compatible.